lib/jsdom/level2/html.js

```diff
@@ -1270,6 +1270,14 @@ define('HTMLLegendElement', {
   proto: {
     get form() {
       return closest(this, 'FORM');
+    },
+    click: function() {
+      if (this.type && this.type === 'submit') {
```

Comment:
Do we need to check `this.type` explicitly? Shouldn't that be cought by the equality check?